sdks/python/hatchet_sdk/runnables/task.py

```diff
@@ -266,7 +270,12 @@ def to_proto(self, service_name: str) -> CreateTaskOpts:
             batch_proto = TaskBatchConfigProto(batch_max_size=self.batch.batch_max_size)
 
             if self.batch.batch_max_interval is not None:
-                interval_ms = int(self.batch.batch_max_interval.total_seconds() * 1000)
+                batch_max_interval = (
+                    str_to_timedelta(self.batch.batch_max_interval)
```

Comment:
not sure I'm following this 😅 why is `self.batch.batch_max_interval` ever a string? shouldn't it be a timedelta?

Reply:
oooh, okay, I see 🤔 

the reason for `Duration` is to allow either a Go duration string (`1s`, `1m`, etc.) _or_ a timedelta, but this was mostly for backwards compat with v0. I think we should only allow `timedelta` here